File: src/main/java/ti/proyectoinia/dtos/SanitarioHongoDto.java
package ti.proyectoinia.dtos;

import lombok.Data;
import ti.proyectoinia.business.entities.TipoSanitarioHongo;

@Data
public class SanitarioHongoDto {

    private Long id;

    private Long sanitarioId;

    private Long hongoId;

    private Integer repeticion;

    private Integer valor;

    private boolean activo;

    private TipoSanitarioHongo tipo;

}

File: frontend/src/models/SanitarioHongo.dto.ts
import { TipoHongoSanitario } from './enums';

export interface SanitarioHongoDTO {
  id: number | null;
  sanitarioId: number | null;
  hongoId: number | null;
  repeticion: number | null;
  valor: number | null;
  incidencia: number | null;
  activo: boolean;
  tipo: TipoHongoSanitario | null;
}
